### PR TITLE
[Docs] Update tflkit README.md

### PR DESCRIPTION
**Pull request by MoonkiHong**
Corresponding to issue #2962, current document in tflkit triggers the Page 404 errors for linking to the 4 example templates. We would like to propose the resolution for this error.

### CODE REVIEW

File: tools/tflkit/README.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ TensorFlow provides some kinds of converting guideline. In Python, the [TFLiteCo
...
+The tflkit uses the [tflite_convert](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/python/tflite_convert.py) python command line interface to convert a TensorFlow model into TfLite model. It only supports to convert a TensorFlow GraphDef file into `TFLITE` format file. This tool supports the creation of individual `TFLITE` files for different input shapes. When converting to multiple `TFLITE` files, it needs to put a string called `NAME` in `TFLITE_PATH`. The string `NAME` will be replaced by what is listed in teh `NAME` environment. This tool requires an information file as a parameter. There is an [example file](convert.template) for a convert information. The `--tensorflow_path` and `--tensorflow_version` can change the TensorFlow location. By default, it uses `externals/tensorflow` directory.
```
Comment (llFreetimell) [inline tools/tflkit/README.md:121]: Simple typo included..!

```suggestion
The tflkit uses the [tflite_convert](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/python/tflite_convert.py) python command line interface to convert a TensorFlow model into TfLite model. It only supports to convert a TensorFlow GraphDef file into `TFLITE` format file. This tool supports the creation of individual `TFLITE` files for different input shapes. When converting to multiple `TFLITE` files, it needs to put a string called `NAME` in `TFLITE_PATH`. The string `NAME` will be replaced by what is listed in the `NAME` environment. This tool requires an information file as a parameter. There is an [example file](convert.template) for a convert information. The `--tensorflow_path` and `--tensorflow_version` can change the TensorFlow location. By default, it uses `externals/tensorflow` directory.
```

**Comment by MoonkiHong** [comment]
@llFreetimell Please let me know about the recent commit suggestion applying your opinion.

**Comment by MoonkiHong** [comment]
@lemmaa Please let me know if the PR is okay. :)

**Comment by MoonkiHong** [comment]
@lemmaa Please let me know if there is further thing that I have to do for this PR. Thank you in advance. :)

**Pull request merged**